File: Proyecto/server/EntityLayer/ProcesoEntity.py
from datetime import datetime
from pydantic import BaseModel
from Utilidades.Enumerado.ProcessActionEnum import ProcessActionEnum

class ProcesoSaveModel(BaseModel):
    ProcesoId: int 
    ModuloSistemaId: int 
    TipoProcesoId: int 
    Nombre: str 
    Descripcion: int 
    EsPordefecto: bool 
    FechaRegistro: datetime 
    CodUsuario: str 
    EstadoRegistro: bool 
    Action: ProcessActionEnum

class ProcesoItemModel:
    ProcesoId: int 
    ModuloSistemaId: int 
    TipoProcesoId: int 
    Nombre: str 
    Descripcion: int 
    EsPordefecto: bool 
    FechaRegistro: datetime 
    CodUsuario: str 
    EstadoRegistro: bool 

    def Cargar(_DB):
        c =  ProcesoItemModel()
        c.ProcesoId = _DB["ProcesoId"] 
        c.ModuloSistemaId = _DB["ModuloSistemaId"] 
        c.TipoProcesoId = _DB["TipoProcesoId"] 
        c.Nombre = _DB["Nombre"] 
        c.Descripcion = _DB["Descripcion"] 
        c.EsPordefecto = bool(ord(_DB["EsPordefecto"])) 
        c.FechaRegistro = _DB["FechaRegistro"] 
        c.CodUsuario = _DB["CodUsuario"] 
        c.EstadoRegistro = bool(ord(_DB["EstadoRegistro"])) 
        return c

File: ProyectoMysql/Api/LogisticStorage/LogisticStorage.EntityLayer/ProcesoEntity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace LogisticStorage.EntityLayer
{
    public class ProcesoEntity: BaseEntityObject
    {

        public ProcesoEntity()
        {
            this.ProcesoId = 0;
            this.ModuloSistemaId = 0;
            this.TipoProcesoId = 0;
            this.Nombre = String.Empty;
            this.Descripcion = String.Empty;
            this.EsPordefecto = false;
            this.FechaRegistro = DateTime.Now;
            this.CodUsuario = String.Empty;
            this.EstadoRegistro = false;
        }


        public Int32 ProcesoId { get; set; }
        public Int32 ModuloSistemaId { get; set; }
        public Int32 TipoProcesoId { get; set; }
        public String Nombre { get; set; }
        public String Descripcion { get; set; }
        public Boolean EsPordefecto { get; set; }
        public DateTime FechaRegistro { get; set; }
        public String CodUsuario { get; set; }
        public Boolean EstadoRegistro { get; set; }
    }
}
